src/context/MassetsProvider.ts

```diff
@@ -21,24 +21,29 @@ const massets: Masset[] = [
       address: process.env.REACT_APP_MUSD_SAVINGS_ADDRESS as string,
     },
   },
-  // TODO enable mBTC
-  // {
-  //   name: 'mBTC',
-  //   address: process.env.REACT_APP_MBTC_ADDRESS as string,
-  //   savingsContract: {
-  //     address: process.env.REACT_APP_MBTC_SAVINGS_ADDRESS as string,
-  //   },
-  // },
+  {
+    name: 'mBTC',
+    address: process.env.REACT_APP_MBTC_ADDRESS as string,
+    savingsContract: {
+      address: process.env.REACT_APP_MBTC_SAVINGS_ADDRESS as string,
+    },
+  },
 ];
 
+const retrievedMasset = localStorage.getItem('selectedMasset');
+
 const initialState: State = {
-  selectedMasset: massets[0],
   massets,
+  selectedMasset: retrievedMasset ? JSON.parse(retrievedMasset) : massets[0],
 };
 
 const reducer: Reducer<State, Action> = (state, action) => {
   switch (action.type) {
     case Actions.SelectMasset: {
+      localStorage.setItem(
```

Comment:
This shouldn't be necessary, and reducers should be pure functions (no side effects). It also bypasses the `LocalStorage` global.